_security/authentication-backends/saml.md

```diff
@@ -185,7 +200,7 @@ Currently, the security plugin supports only the `HTTP-Redirect` logout binding.
 
 ## Exchange key settings
 
-SAML, unlike other protocols, is not meant to be used for exchanging user credentials with each request. The security plugin trades the SAML response for a lightweight JSON web token that stores the validated user attributes. This token is signed by an exchange key that you can choose freely. Note that when you change this key, all tokens signed with it become invalid immediately.
+SAML, unlike other protocols, is not meant to be used for exchanging user credentials with each request. The security plugin trades the SAML response for a lightweight JSON Web Token that stores the validated user attributes. This token is signed by an exchange key that you can choose freely. Note that when you change this key, all tokens signed with it become invalid immediately.
 
 Name | Description
```

Comment:
To prevent any table rendering issues, I would add a pipe before "Name" and after "Description", and the justification on the next line.